advisor/src/main/kotlin/advisors/Osv.kt

```diff
@@ -185,13 +185,12 @@ private fun Vulnerability.toOrtVulnerability(): org.ossreviewtoolkit.model.Vulne
         }
     } ?: (null to null)
 
-    if (severity == null && databaseSpecific != null) {
+    val specificSeverity = databaseSpecific?.get("severity")
```

Comment:
Consider following alternative which avoid the variable `specificSeverity`:

```
if (severity == null) {
   databaseSpecific?.get("severity")?.let {
     if (it is JsonPrimitive) {
         severity = it.contentOrNull
     }
   }
}
```

Reply:
I prefer to keep the current code as it makes clear already in the outer condition to only do something if we don't have a `severity` *but* we have a `specificSeverity`.